scico/linop/radon_svmbir.py

```diff
@@ -190,20 +192,28 @@ class SVMBIRWeightedSquaredL2Loss(WeightedSquaredL2Loss):
     of :class:`scico.linop.Diagonal`. If :math:`W` is None, it is set to
     :class:`scico.linop.Identity`.
 
+    The extended loss differs from a typical weighted squared :math:`\ell_2` loss
+    is the following ways.
```

Comment:
in the following ways?

Reply:
How about "... in the following aspects"?

Reply:
Changed it.